testing/matrix-sdk-test/src/test_json/keys_query_sets.rs

```diff
@@ -46,61 +259,21 @@ impl KeyDistributionTestData {
     pub const USER_SIGNING_KEY_PRIVATE_EXPORT: &'static str =
         "zQSosK46giUFs2ACsaf32bA7drcIXbmViyEt+TLfloI";
 
+    /// Current user's private user-signing key, as an [`Ed25519SecretKey`].
+    pub fn me_private_user_signing_key() -> Ed25519SecretKey {
+        Ed25519SecretKey::from_base64(Self::USER_SIGNING_KEY_PRIVATE_EXPORT).unwrap()
+    }
```

Comment:
we end up using this in a few places in a later commit, hence pulling it out to a separate function